### PR TITLE
[#50] Width Feature

### PR DESCRIPTION
**Pull request by xeoh**
see [#50].

### CODE REVIEW

File: app/src/main/java/kr/ac/kaist/team888/locator/Locator.java [hunk 1]
```diff
@@ -354,11 +360,28 @@ private void applyContour() {
...
+  private void applyWidth() {
+    double width = (WIDTH_MAX - WIDTH_MIN) * FeatureController.getInstance().getWidth() + WIDTH_MIN;
+    double totalWidth = (HangulCharacter.ORIGIN_REGION.getMaxX()
+        - HangulCharacter.ORIGIN_REGION.getMinX()) * width;
+    double widthCenter = HangulCharacter.ORIGIN_REGION.getMaxX()
+        - HangulCharacter.ORIGIN_REGION.getMinX() / 2;
+    locatorRegion.setMinX(widthCenter - totalWidth / 2);
+    locatorRegion.setMaxX(widthCenter + totalWidth / 2);
+
+    for (ArrayList<BezierCurve> segmentData : skeletons) {
```
Comment (hangpark) [inline app/src/main/java/kr/ac/kaist/team888/locator/Locator.java]: Change `segmentData` to `skeleton`.

File: app/src/main/java/kr/ac/kaist/team888/locator/Locator.java [hunk 2]
```diff
@@ -354,11 +360,28 @@ private void applyContour() {
...
+      for (int i = 0; i < segmentData.size(); i++) {
```
Comment (hangpark) [inline app/src/main/java/kr/ac/kaist/team888/locator/Locator.java]: Change `segmentData` to `skeleton`.

File: app/src/main/java/kr/ac/kaist/team888/locator/Locator.java [hunk 3]
```diff
@@ -354,11 +360,28 @@ private void applyContour() {
...
+        segmentData.set(i,
```
Comment (hangpark) [inline app/src/main/java/kr/ac/kaist/team888/locator/Locator.java]: Change `segmentData` to `skeleton`.

**Review by hangpark — CHANGES_REQUESTED** [review]
`segment` is used for indicating a single chunk of curves from a joint to a next joint.

**Review by bimak-server — APPROVED** [review]
👍

**Review by hangpark — APPROVED** [review]

**Pull request merged**